cmd/kubelet/app/options/options.go

```diff
@@ -196,7 +196,7 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 	if f.DynamicConfigDir.Provided() && !utilfeature.DefaultFeatureGate.Enabled(features.DynamicKubeletConfig) {
 		return fmt.Errorf("the DynamicKubeletConfig feature gate must be enabled in order to use the --dynamic-config-dir flag")
 	}
-	// ensure that nobody sets InitConfigDir if the dynamic config feature gate is turned off
+	// ensure that nobody sets InitConfigDir if the KubeletConfigFile feature gate is turned off
```

Comment:
The change looks fine to me and this is already got lgtm. Let's ping @kubernetes/sig-node-pr-reviews for approval.